docs/en/stack/ml/nlp/ml-nlp-model-ref.asciidoc

```diff
@@ -97,3 +185,26 @@ Using `DPREncoderWrapper`:
 * https://huggingface.co/valhalla/distilbart-mnli-12-6
 * https://huggingface.co/cross-encoder/nli-distilroberta-base
 * https://huggingface.co/cross-encoder/nli-roberta-base
+
+[discrete]
+=== Zero-shot text classification expected model output
+
+Zero-shot text classification allows text to be classified for arbitrary labels not necessarily part of the original
+training. Each sequence is combined with the label given some hypothesis template. The model then scores each of these
+combinations according to `[entailment, neutral, contradiction]`. The output of the model must be a float tensor 
+with `shape(<number of sequences>, <number of labels>, <3>)`.
```

Comment:
If the angular brackets are intended to denote variable values and 3 is not a variable, I think we should remove the brackets:
```suggestion
with `shape(<number of sequences>, <number of labels>, 3)`.
```